server/src/main/java/nl/inl/blacklab/server/requesthandlers/RequestHandler.java

```diff
@@ -456,6 +455,16 @@ private static boolean doDebugSleep(HttpServletRequest request) {
 
     }
 
+    private void setRequestIds() {
+        String requestId= request.getHeader(ANN_REQUEST_ID_HEADER_NAME);
+        if (requestId == null) {
+            requestId = "unknown";
+        }
+        String b64uuid = Base64.getUrlEncoder().encodeToString(UUID.randomUUID().toString().getBytes());
```

Comment:
I may have missed some earlier discussion about this, but why not have the rule id replace this UUID? Based on the log sample you pasted it looked like there's a 1-1 correspondence there. Just thinking about ways to keep the logs tight.

Reply:
That could work too